Makefile

```diff
@@ -67,6 +67,16 @@ DEFS += -DNDEBUG
 DEFS += -DCONFIG_NFCT_PINS_AS_GPIOS
 DEFS += -DBUILD_VERSION='"$(BUILD_VERSION)"'
 DEFS += -DLFS2_NO_ASSERT
+DEFS += -DFIXED_POINT
+DEFS += -DUSE_KISS_FFT
```

Comment:
There are several FFT implementations we can choose from for performance tuning.